### PR TITLE
Add tests to verify access to the dashboard settings and Usage Data Collection

### PR DESCRIPTION
**Pull request by jgarciao**
- ODS-1216 - Verify that users in group rhods-admin can access the "Cluster Settings" menu
- ODS-1217 - Verify that users not in group rhods-admin cannot access the "Cluster Settings" menu
- ODS-1218 - Verify that "Usage Data Collection" can be set in the "Cluster Settings" menu
- ODS-1234 - Verify that "Usage Data Collection" is enabled by default
- ODS-1235 - Verify tracking key used for "Usage Data Collection"

### CODE REVIEW

File: tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot [hunk 1]
```diff
@@ -339,3 +340,24 @@ Check External Links Status
...
+    Page Should Contain    Settings
+    Menu.Navigate To Page    Settings    Cluster settings
+    Capture Page Screenshot
+    Wait Until Page Contains    Update global settings for all users    timeout=30
+    Wait Until Page Contains Element    ${USAGE_DATA_COLLECTION_XP}    timeout=30
+
+Verify Cluster Settings Is Not Available
+    [Documentation]    Verifies submenu Settings > Cluster settings is not visible
+    ${cluster_settings_available}=    Run Keyword And Return Status    Verify Cluster Settings Is Available
+    Should Be True    not ${cluster_settings_available}    msg=Cluster Settings shoudn't be visible for this user
```
Comment (bdattoma) [inline tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot:355]: would it be possible to use `Should Not Be True` to be consistent with what you did in ODS keywords

File: tests/Tests/400__ods_dashboard/410__ods_dashboard_settings.robot [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    [Documentation]    Opens a browser and logs into ODS Dashboard with a user belonging to the rhods-admins group
+    Open Browser    ${ODH_DASHBOARD_URL}    browser=${BROWSER.NAME}    options=${BROWSER.OPTIONS}
+    Login To RHODS Dashboard    ${TEST_USER.USERNAME}    ${TEST_USER.PASSWORD}    ${TEST_USER.AUTH_TYPE}
+    Wait For RHODS Dashboard To Load
+
+Open ODS Dashboard With Non Admin User
+    [Documentation]    Opens a browser and logs into ODS Dashboard with a user belonging to the rhods-users group
+    Open Browser    ${ODH_DASHBOARD_URL}    browser=${BROWSER.NAME}    options=${BROWSER.OPTIONS}
+    Login To RHODS Dashboard    ${TEST_USER_3.USERNAME}    ${TEST_USER_3.PASSWORD}    ${TEST_USER_3.AUTH_TYPE}
+    Wait For RHODS Dashboard To Load
```
Comment (bdattoma) [inline tests/Tests/400__ods_dashboard/410__ods_dashboard_settings.robot:73]: you could use `Launch Dashboard` keyword

File: tests/Resources/ODS.robot [hunk 3]
```diff
@@ -8,6 +8,10 @@ Resource            ./Page/ODH/JupyterHub/ODHJupyterhub.resource
 Resource            ./Page/ODH/Prometheus/Prometheus.resource
 
 
+*** Variables ***
+${USAGE_DATA_COLLECTION_DEFAULT_SEGMENT_PUBLIC_KEY}     KRUhoAIEpWlGuz4sWixae1vAXKKGlD5K
```
[the remark this answers is not in the document]
Comment (jgarciao) [reply tests/Resources/ODS.robot:12]: Its a public key.  It's in [GitHub already](https://github.com/red-hat-data-services/odh-deployer/blob/30897387d7fbde3c34fc5fb9bfac8b2042226871/monitoring/segment-key-secret.yaml#L9) (in base 64)  and any RHODS user accessing to the dashboard will be able to see it in their browser's console. I think it doesn't need to be hidden

File: tests/Tests/400__ods_dashboard/410__ods_dashboard_settings.robot (hunk 2 above)
Comment (jgarciao) [reply tests/Tests/400__ods_dashboard/410__ods_dashboard_settings.robot:73]: Fixed. Thanks

**Comment by jgarciao** [comment]
It's being tested here: https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/533/

**Review by bdattoma — APPROVED** [review]
lgtm

**Comment by jgarciao** [comment]
Previous job didn't work (no tests were executed). Trying again here:
- https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/535/
- https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/536/

**Pull request merged**